### ISSUE TITLE
white list 

### ISSUE THREAD

**Issue by rowhack** [R-Smith/tcpTrigger#7]
Would it be possible to allow the use of ip ranges also.

I have the situation where I what to white list several subnets.

**Comment by r-smith** [comment]
Yes, I'm working on this now. I'm redesigning the GUI for the whitelist and will allow for ranges for both the IP address and port numbers.

**Comment by rowhack** [comment]
Oh wow, that would be great!
thanks

From: Ryan Smith ***@***.***>
Sent: Thursday, March 30, 2023 2:43 PM
To: r-smith/tcpTrigger ***@***.***>
Cc: Watson, Richard ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL EMAIL]: Re: [r-smith/tcpTrigger] white list (Issue #7)


Yes, I'm working on this now. I'm redesigning the GUI for the whitelist and will allow for ranges for both the IP address and port numbers.
